### ISSUE TITLE
URL design

### ISSUE THREAD

**Issue by DonaldTsang** [DangerOnTheRanger/maniwani#63]
1. Named boards should have matching URLs instead of board IDs
2. Using URL to search board threads would be useful

**Comment by DangerOnTheRanger** [comment]
1 wouldn't be too hard to implement; it would require modifying the `boards` blueprint to get boards by name (in addition to or instead of?) ID, and then reworking existing references to board IDs in the templates to board names. So a lot of files will need to be modified, but the work would be straightforward.

I'm not sure I understand 2. Do you mean being able to have an URL that you can use to search a board?

**Comment by DonaldTsang** [comment]
I recommend doing research on how 4chan and 8chan (especially nerv) works, you might see some inspiration.

**Comment by DangerOnTheRanger** [comment]
Are you referring to multiboards, i.e, being able to do something like view tech/cyber/g/sudo/prog/sci/ simultaneously? That wouldn't be too hard to implement on the backend (the firehose/front page does this already), though I'm not sure how you'd design the frontend outside of having the user type the URL in manually.

**Comment by DonaldTsang** [comment]
Yep. Basic URLs for boards, threads, multiboards and catalog search are some basic features you have to consider.

**Comment by DangerOnTheRanger** [comment]
Searching is probably something better discussed on #59, but I'm open to adding multiboards. Again, the UI is something that should be worked out, but if someone were to submit a PR that only added backend support for multiboards (i.e, code that took an URL and spit out a multiboard, that's it) I would accept it, and work on the UI later.

**Comment by DonaldTsang** [comment]
@DangerOnTheRanger check this again and see if the design is good, and if not, what could be changed.
